tests/test_quickLook.py

```diff
@@ -0,0 +1,313 @@
+            {"instrument": instrument, "physical_filter": physical_filter, "detector": detector})
+        self.detector_id = self.repo.registry.expandDataId(
+            {"instrument": instrument, "detector": detector})
+        self.filter_id = self.repo.registry.expandDataId(
+            {"instrument": instrument, "physical_filter": physical_filter})
+
+        # put empty data
+        transmissionCurve = TransmissionCurve.makeSpatiallyConstant(np.ones(2), np.linspace(0, 1, 2), 0., 0.)
+        self.butler = butlerTests.makeTestCollection(self.repo)
+        self.butler.put(afwImage.ExposureF(), ccdExposure, self.exposure_id)
+        self.butler.put(afwTestUtils.CameraWrapper().camera, camera, self.instrument_id)
+        self.butler.put(afwImage.ExposureF(), bias, self.detector_id)
+        self.butler.put(afwImage.ExposureF(), dark, self.detector_id)
+        self.butler.put(afwImage.ExposureF(), flat, self.flat_id)
+        self.butler.put(lsst.ip.isr.Defects(), defects, self.detector_id)
+        self.butler.put(np.zeros(2), bfKernel, self.instrument_id)
+        self.butler.put(lsst.ip.isr.brighterFatterKernel.BrighterFatterKernel(),
+                        newBFKernel, self.detector_id)
+        self.butler.put(ipIsr.PhotonTransferCurveDataset(), ptc, self.detector_id)
+        self.butler.put(transmissionCurve, filterTransmission, self.filter_id)
+        self.butler.put(lsst.ip.isr.calibType.IsrCalib(), deferredChargeCalib, self.detector_id)
+        self.butler.put(transmissionCurve, opticsTransmission, self.instrument_id)
+        self.butler.put(afwImage.ExposureF(), strayLightData, self.flat_id)
+        self.butler.put(transmissionCurve, atmosphereTransmission, self.instrument_id)
+        self.butler.put(lsst.ip.isr.crosstalk.CrosstalkCalib(), crosstalk, self.detector_id)
+        self.butler.put(afwImage.ExposureF().maskedImage, illumMaskedImage, self.flat_id)
+        self.butler.put(lsst.ip.isr.linearize.Linearizer(), linearizer, self.detector_id)
+        self.butler.put(afwImage.ExposureF(), fringes, self.flat_id)
+        self.butler.put(transmissionCurve, sensorTransmission, self.detector_id)
+        self.butler.put(afwImage.ExposureF(), crosstalkSources, self.exposure_id)
+
+    def tearDown(self):
+        del self.repo_path  # this removes the temporary directory
+
+    def test_runQuantum(self):
+        config = ipIsr.IsrTaskConfig()
+        # Remove some outputs
+        config.doBinnedExposures = False
+        config.doSaveInterpPixels = False
+        config.qa.doThumbnailOss = False
+        config.qa.doThumbnailFlattened = False
+        config.doCalculateStatistics = False
+
+        # Turn on all optional inputs
+        config.doAttachTransmissionCurve = True
+        config.doIlluminationCorrection = True
+        config.doStrayLight = True
+        config.doDeferredCharge = True
+        config.usePtcReadNoise = True
+        config.doCrosstalk = True
+        config.doBrighterFatter = False
```

Comment:
I'm a little confused by this block in two small ways: it says to turn on all option inputs, but it doesn't include many (mostly the main ones, like bias/dark/flat etc), and also `config.doBrighterFatter` is set to `False` whereas all the others are `True`. Could you just clarify?

Reply:
Good catch! I forgot to turn this back on after I got BF working.